src/app/components/editable-markdown/editable-markdown.component.html

```diff
@@ -3,7 +3,11 @@
     <markdown [data]="markdownText" emoji></markdown>
   </section>
   <br>
+<<<<<<< HEAD
+  <button (click)="toggleEditor()">{{ 'Modifier le contenu de cette page' | translate }}</button>
+=======
```

Comment:
merge conflict a corriger

Reply:
@sidiwhynot if you got time fixe these and other comments so i can merge your work